focus_converter_base/focus_converter/conversion_configs/aws/mapping_files/aws_category_mapping.csv

```diff
@@ -1,4 +1,4 @@
-product_code,aws_category
+line_item_product_code,ServiceCategory
```

Comment:
If we are referencing source column name in mappings, then would it make sense to keep separate copy of category mapping for aws-cur ? 
So we can keep a mapping file  -
focus_converter_base/focus_converter/conversion_configs/aws-cur/mapping_files/aws_cur_category_mapping.csv
This way we can keep actual source column names for each type to avoid any conflicts.

@brianwyka @varunmittal91  please let me know your thoughts on this.

Reply:
@spriharani thank you for bringing that up, I missed that in my own review. 

I think that is a good point, although the issue then would be that we would have to maintain two separate copies. Which is why I was wondering, for such cases if there was a way for us to reduce cost data from aws, and aws-cur into a common denomination and then proceed with conversion. Although this can be a later thing and for now we do whatever is faster.